Makefile

```
@@ -16,5 +16,5 @@ release: clean
 
 force_release: clean
 	git push --tags origin main
-	python3 setup.py sdist bdist_wheel
+	python -m build --sdist --wheel
```

By default, both are built:

```suggestion
	python -m build 
```

---

Thanks for the suggestion (at least *I* was unaware). In this particular case though, I like keeping it explicit, and for it to remain similar to its previous state.